osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -102,6 +108,19 @@ protected override void UpdateAfterChildren()
             base.UpdateAfterChildren();
 
             comboDisplay.X = Catcher.X;
+
+            if (!lastHyperDashState && Catcher.HyperDashing && Time.Elapsed > 0)
+                catcherTrails.DisplayEndGlow(Catcher.CurrentState, Catcher.X, Catcher.BodyScale);
```

Comment:
The behaviour hasn't changed (neither has naming), but I find it weird that this is call "end glow" when it's at the start of the hyperdash. Almost reads like a bug.

Reply:
I agree. I kept this PR single scoped, but I wanted to fix this naming inconsistency.